HomeWorks/HW2/Bor/Tire.cs

```diff
@@ -0,0 +1,121 @@
+namespace TrieHomework;
+
+using System.Collections.Generic;
+
+public class Trie
+{
+    private class Node
+    {
+        public Dictionary<char, Node> Children { get; } = new();
+        public bool IsEnd { get; set; }
+        public int Count { get; set; }
+    }
+
+    private readonly Node _root = new();
+    private int _size;
+
+    /// <summary>
+    /// Gets the number of unique elements stored in the trie.
+    /// </summary>
+    public int Size => _size;
```

Comment:
Почему бы не использовать [автоматическое свойство](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/properties#automatically-implemented-properties)?
```suggestion
    /// <summary>
    /// Gets the number of unique elements stored in the trie.
    /// </summary>
    public int Size { get; private set; }
```